charts/cloudquery/templates/cronjob.yaml

```diff
@@ -32,9 +32,15 @@ spec:
               resources:
               {{- toYaml .Values.resources | nindent 16 }}
               volumeMounts:
+              {{- if .Values.volumeMounts }}
+              {{- toYaml .Values.volumeMounts | nindent 14 }}
```

Comment:
shouldn't it be 16 as in `resources` above?

Reply:
Same as above.